ARCore/README.md

```diff
@@ -0,0 +1,4 @@
+# TokBox integration with Android ARCore
+This sample code will help you to better understand in integrating `opentok` Android SDK with `ARCore` from Google. To make this sample code running, you need to fill up some information inside `AugmentedFaceActivity`.
+
+Please fill `TOKEN`, `SESSION_ID` and `API_KEY`
```

Comment:
Can you please include some basic description of how this sample works? It doesn't have to be much, but it should include snippets or pointers to the code that uses ARCore.  Perhaps a short description of how the AugmentedFaceActivity class implements ARCore. Here's an example: https://github.com/opentok/opentok-android-sdk-samples/tree/master/Custom-Audio-Driver.

Reply:
Sure